backend/typescript/services/implementations/emailService.ts

```diff
@@ -0,0 +1,35 @@
+import nodemailer, { Transporter } from "nodemailer";
+import IEmailService from "../interfaces/emailService";
+import { NodemailerConfig } from "../../types";
+import Logger from "../../utilities/logger";
+
+class EmailService implements IEmailService {
+  transporter: Transporter;
+
+  constructor(nodemailerConfig: NodemailerConfig) {
+    this.transporter = nodemailer.createTransport(nodemailerConfig);
+  }
+
+  async sendEmail(
+    from: string,
```

Comment:
Minor thing, but maybe this should be set in the constructor as well?

Reply:
Another thought, instead of taking the entire email address as a constructor argument, we can just take the display name since the email address is already in the nodemailer config. This way we have complete control over the format of the from address (don't have the rely on users giving us "Display Name <email address>" exactly)